kafka/consumer/multiprocess.py

```diff
@@ -105,7 +108,8 @@ class MultiProcessConsumer(Consumer):
     def __init__(self, client, group, topic, auto_commit=True,
                  auto_commit_every_n=AUTO_COMMIT_MSG_COUNT,
                  auto_commit_every_t=AUTO_COMMIT_INTERVAL,
-                 num_procs=1, partitions_per_proc=0):
+                 num_procs=1, partitions_per_proc=0,
+                 simple_consumer_options=None):
```

Comment:
I think this should be `**simple_consumer_options` to keep kwargs interface simple and consistent.  agree?